README.md

```diff
@@ -1 +1,29 @@
-Documentation resides in the [wiki](https://github.com/NREL/alfalfa/wiki)!
+# Alfalfa
+
```

Comment:
:)

Reply:
@anyaelena  -- can you review this README before we merge into Toby's branch?